base_rest/models/rest_service_registration.py

```diff
@@ -49,6 +58,52 @@ def _register_hook(self):
             (name, cls) for name, cls in controllers if "RestController" not in name
         ]
         http.controllers_per_module["base_rest"] = controllers
+        # create the final controller providing the http routes for
```

Comment:
I am testing this new version of rest api, and it's seem that we have some issue here.
If I stop/start odoo and refresh quickly odoo in my web browser it fail to load the registry correctly and my server is crashed for ever, so I need to start and wait some second.
Also If I run the base_rest_demo/test/test_openapi.py I have the issue with the registry. I am not sure that it's related.

@lmignon can you try in local to see if you have the same issue?

Reply:
It's seem that I do not have the issue anymore.